routing/go.mod

```diff
@@ -0,0 +1,26 @@
+module cloud.google.com/go/routing
```

Comment:
I bet we will eventually be generating more maps APIs. I think we should make the module under `cloud.google.com/go/maps` and have routing be a subdir

Reply:
Great, will do.

Reply:
Fixed, thanks!